android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -598,6 +597,14 @@ private void doMenuAction(int menuItemId) {
                 }
                 break;
         }
+
+        if(menuItemId != R.id.nav_schedule
```

Comment:
In our case set is {b, c, d, e, f, g, h, i, j}
So what will be better `( a!= b && a!=c && a!=d && a!=e)` or `(a==f || a==g || a=h || a==i || a==j)` ?